src/test/java/net/openhft/chronicle/queue/QueueTestCommon.java

```diff
@@ -131,7 +131,7 @@ public void recordExceptions() {
         exceptionTracker = JvmExceptionTracker.create(false);
         if (OS.isWindows())
             ignoreException("Read-only mode is not supported on Windows® platforms, defaulting to read/write");
-        for (String msg : "Shrinking ,Allocation of , ms to add mapping for ,jar to the classpath, ms to pollDiskSpace for , us to linearScan by position from ,File released ,Overriding roll length from existing metadata, was 3600000, overriding to 86400000   ".split(",")) {
+        for (String msg : "Forced unlock, Shrinking ,Allocation of , ms to add mapping for ,jar to the classpath, ms to pollDiskSpace for , us to linearScan by position from ,File released ,Overriding roll length from existing metadata, was 3600000, overriding to 86400000   ".split(",")) {
```

Comment:
I don't think this should be a blanket addition. It should not happen too often in the tests surely?

Reply:
Good call Jerry - that's changed